docs/source/dev_guide/review_process.rst

```diff
@@ -0,0 +1,95 @@
+.. _review_process:
+
+.. contents::
+   :local:
+
+Review process
+--------------
+
+This short description should serve as a rough guideline on how we deal with
+reviews on pull requests (PR). Some PRs are easy, others are hard. Some require
+multiple changes and continuous feedback, others can be merged in
+flawlessly. The purpose of this text is to communicate our intent and style and
+should not be seen as *law*.
```

Comment:
Add: "A summary of these guidelines can be found in `CONTRUBTING.rst` in the top level of the repository. If you haven't yet read that file, it would be a good idea to do that now."